ipatests/test_integration/test_installation.py

```diff
@@ -1108,6 +1108,74 @@ def run_request(url, expected_stdout=None, expected_stderr=None):
                 expected_stdout=f'href="https://{self.master.hostname}/'
             )
 
+    def test_hostname_parameter(self):
+        """
+        Test for #2692 ipa-server-install ignores --hostname
+
+        https://pagure.io/freeipa/issue/2692
+        """
+        original_hostname = self.master.hostname
+        new_hostname = "ipa2692.test"
```

Comment:
Single label domain. It is probably better to use the existing domain just a new hostname in it.